server/auth.go

```diff
@@ -764,6 +765,38 @@ func checkClientTLSCertSubject(c *client, fn tlsMapAuthFn) bool {
 	return false
 }
 
+func dnsAltNameLabels(dnsAltName string) []string {
+	return strings.Split(strings.ToLower(dnsAltName), ".")
+}
+
+// Check DNS name according to https://tools.ietf.org/html/rfc6125#section-6.4.1
+func dnsAltNameMatches(dnsAltNameLabels []string, urls []*url.URL) bool {
+URLS:
+	for _, url := range urls {
+		if url == nil {
+			continue URLS
```

Comment:
What is your intent here? Moving to the next url? If so, you don't need the `URLS:` label and `continue` alone will just do that.

Reply:
Yes, I added it primarily because I need the label in one place, so I opted to provide it wherever it applies.
remove where unnecessary?

Reply:
I would.

Reply:
removed